python/tvm/relay/op/contrib/arm_compute_lib.py

```diff
@@ -227,9 +229,10 @@ def _func_wrapper(attrs, args):
     return _func_wrapper
 
 
+# Reshape does not need padding check in 20.05
 _register_external_op_helper("reshape")
 
-
+# conv2d does not need padding check in 20.05
```

Comment:
I cannot link this comment with the code.

Reply:
removed